### DISCUSSION TITLE
Wallpaper

### DISCUSSION THREAD

**Discussion by Mikrizi** [evilsocket/opensnitch#1038] [General]
Greetings, can you please share this wallpaper on the main page of the program? I can not find.
https://user-images.githubusercontent.com/2742953/85205382-6ba9cb00-b31b-11ea-8e9a-bd4b8b05a236.png

**Comment by gustavo-iniguez-goya** [comment]
hey @Mikrizi ,

Here it is: [mars wallpaper](https://github.com/evilsocket/opensnitch/assets/2742953/0128abf0-e9e8-4e72-8feb-5d808ffd475e)

I don't remember where I downloaded it from, and it seems like it no longer exists online.
